packages/react-accordion/package.json

```diff
@@ -17,16 +17,18 @@
     "code-style": "just-scripts code-style",
     "just": "just-scripts",
     "lint": "just-scripts lint",
-    "start": "just-scripts dev:storybook",
-    "start-test": "just-scripts jest-watch",
-    "test": "just-scripts test",
-    "update-snapshots": "just-scripts jest -u"
+    "start": "storybook",
+    "test": "jest",
+    "docs": "api-extractor run --config=config/api-extractor.local.json --local",
+    "build:local": "tsc -p . --module esnext --emitDeclarationOnly && node ../../scripts/typescript/normalize-import --output dist/packages/react-accordion/src && yarn docs",
+    "storybook": "start-storybook"
   },
   "devDependencies": {
     "@fluentui/babel-make-styles": "^9.0.0-alpha.15",
     "@fluentui/eslint-plugin": "^1.3.1",
     "@fluentui/jest-serializer-make-styles": "^9.0.0-alpha.17",
     "@fluentui/react-conformance": "^0.4.1",
+    "@fluentui/react-icons-mdl2": "^1.1.2",
```

Comment:
you should get rid of this and manually add those icons as needed within your stories . check react menu for example -> https://github.com/microsoft/fluentui/blob/master/packages/react-menu/src/Menu.stories.tsx#L18

Reply:
Ok, I'll do that for now but it seems like we're duplicating temporary solutions https://github.com/microsoft/fluentui/blob/master/packages/react-menu/src/tmp-icons.stories.tsx#L2
Let's make an Issue to solve that once and for all 😉